skops/card/_model_card.py

```diff
@@ -10,6 +10,7 @@
 from reprlib import Repr
 from typing import Any, Optional, Union
 
+import matplotlib.pyplot as plt
```

Comment:
I don't think we can do that, since matplotlib is not an install dependency. We could add it but I'd prefer not to. So for this feature to still work, we would need to make the import local (i.e. in the method itself), which is also not super nice. @adrinjalali what's your opinion?

Reply:
Yes, better than just failing when the import fails

Reply:
Yes, we shouldn't import in the module level, we should just try to import in the method, and raise a meaningful error if it fails.

Reply:
I just edited it 🙂